Firestore/Example/Tests/Remote/FSTSerializerBetaTests.mm

```diff
@@ -486,6 +486,18 @@ - (void)testEncodesRelationFilter {
   XCTAssertEqualObjects(actual, expected);
 }
 
+- (void)testEncodesArrayContainsFilter {
+  FSTRelationFilter *input = FSTTestFilter("item.tags", @"array_contains", @"food");
+  GCFSStructuredQuery_Filter *actual = [self.serializer encodedRelationFilter:input];
+
+  GCFSStructuredQuery_Filter *expected = [GCFSStructuredQuery_Filter message];
+  GCFSStructuredQuery_FieldFilter *prop = expected.fieldFilter;
+  prop.field.fieldPath = @"item.tags";
+  prop.op = GCFSStructuredQuery_FieldFilter_Operator_ArrayContains;
+  prop.value.stringValue = @"food";
+  XCTAssertEqualObjects(actual, expected);
```

Comment:
No action required, but just to confirm: if you did an item.tags array_contains "food" and item.animals array_contains cats that encodes as fully separate FieldFilters, right?

Reply:
Yes (and I confirmed via logging just to be 100% certain).